### PR TITLE
Add Mouse Jiggler to HID Remote

### PR DESCRIPTION
**Pull request by MrLemur**
# What's new

- Adds a new remote "Mouse Jigggler" for USB and Bluetooth. Moves the mouse back and forth when started (inspired from this [repo](https://github.com/jplesner/flipperzero-firmware-mouse-jiggler))
- Tries to use same functions as in existing mouse  app

# Verification 

- Either on USB or Bluetooth remote, run the remote and press Start.

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/plugins/hid_app/views/hid_mouse_jiggler.c [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        elements_multiline_text_aligned(canvas, 91, 36, AlignLeft, AlignBottom, "Start");
+    }
+    canvas_set_color(canvas, ColorBlack);
+
+    // Back
+    canvas_draw_icon(canvas, 74, 49, &I_Pin_back_arrow_10x8);
+    elements_multiline_text_aligned(canvas, 91, 57, AlignLeft, AlignBottom, "Quit");
+
+    if(running) {
+        hid_hal_mouse_move(parentJiggler->hid, MOUSE_MOVE_SHORT, 0);
```
Comment (skotopes) [inline applications/plugins/hid_app/views/hid_mouse_jiggler.c]: View/ViewPort draw call is meant only for drawing, no API calls allowed in its context.
Please rewrite this code in a way when hid_hall only called from application main thread.

**Review by skotopes — CHANGES_REQUESTED** [review]

**Comment by skotopes** [comment]
Please un-draft when you finish.

File: applications/plugins/hid_app/views/hid_mouse_jiggler.c (hunk 1 above)
Comment (MrLemur) [reply applications/plugins/hid_app/views/hid_mouse_jiggler.c]: Thanks for the comment! As I am trying to use the existing HID app, I have not seen any examples where a function can run continously while not locking the main thread and allowing input callbacks.

Is there something like a event loop that can be used in addition to the main thread, or would it be better to move this to its own app and logic?

Comment (skotopes) [reply applications/plugins/hid_app/views/hid_mouse_jiggler.c]: You can use furi_timer and do the work in timer callback

Comment (MrLemur) [reply applications/plugins/hid_app/views/hid_mouse_jiggler.c]: Ah fantastic! Thank you!

**Comment by MrLemur** [comment]
> Please un-draft when you finish.

Changed to use furi_timer

**Review by skotopes — APPROVED** [review]

**Comment by MrLemur** [comment]
@skotopes Thank you for the refactor! This is my first time using C coming from Golang, so great to see how to effectively use the Furi API.

**Pull request merged**